wallet-adapter/yarn.lock

```diff
@@ -0,0 +1,9667 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We should keep only package-lock.json, (since It seems that this is the only yarn.lock, other packages use npm), let's remove this.

Having both files/ using both npm and yarn can create inconsistencies

Reply:
removed!